exercises/concept/rpn-calculator/src/lib.rs

```diff
@@ -0,0 +1,19 @@
+// This stub file contains items which aren't used yet; feel free to remove this module attribute
+// to enable stricter warnings.
+#![allow(unused)]
```

Comment:
Please remove this. The typical pattern we encourage is to put function parameters into the `unimplemented` section to ensure they're used, or worst case, to prepend a `_` to the name. `#[allow(unused)]` for the whole module is too broad.

Reply:
That was included because of the CI was not liking the enum. Now that I think of it, it may have been because I forgot to make it `pub`.

Fixed in c413d4e